### DISCUSSION TITLE
Tutorial map text needs to be updated to reflect that Notes "tab" has been removed from 2.6

### DISCUSSION THREAD

**Discussion by tvleavitt** [triplea-game/triplea#12115] [General]
I just downloaded the latest 2.6 build onto a new iMac, and loaded up the Tutorial map. Two things:

a) The tutorial documentation (first page of description in Maps selection) should explicitly tell people how to start a game (with screen shots), and to select "Default" when presented with "Set All To" before the game is actually loaded (why isn't "Default" pre-selected in the UI to begin with)?

b) More importantly... there's no "Notes" tab in the main window UI (any longer?); it appears to have been moved to a menu item under "Help"; 99% of first-time users are going to be utterly baffled by this, thus rendering the Tutorial (aimed at newbies) basically useless as a tool for introducing people to the game.

**Comment by DanVanAtta** [comment]
All of these updates look like they would be part of the tutorial map itself: https://github.com/triplea-maps/tutorial

@tvleavitt do you know offhand where is the confusing mention about notes? The notes has previously existed in two places, now it is just in the one.

**Reply by DanVanAtta** [reply]
Correction, the map thumbnail for 2.5 would need to be fixed in triplea_maps.yaml, in 2.6; the game-notes are pulled from the map.yml file that would be contained in the map repo itself.

**Comment by tvleavitt** [comment]
This is only the first reference... see embedded screencap with pointer, too. My take is that the Tutorial should be systematically reviewed for the impact of changes made for 2.6 prior to the release of said version. Or, removed (not recommended). I know this means more maintenance effort, but I view an accurate, up-to-date, and functional Tutorial map as an essential requirement.

![2023-11-25 15_10_24-Notification](https://github.com/triplea-game/triplea/assets/4445057/48815ee4-3adf-4b0b-bd1c-538804b0e3cf)
